client/src/components/SignIn.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import axios from 'axios';
+
+import { useInput } from '../utilities/useInput';
+
+const SignIn = ({ history }) => {
+  const usernameSignIn = useInput();
+  const passwordSignIn = useInput();
+
+  const signIn = async e => {
```

Comment:
everything coming together from async/await in this unit with refined hook usage form React -- very well done Chris